src/main/java/winstone/HostConfiguration.java

```diff
@@ -177,7 +177,7 @@ public void preConfigure() throws Exception {
                 }
                 setMaxFormKeys(Option.MAX_PARAM_COUNT.get(args));
                 setMaxFormContentSize(Option.REQUEST_FORM_CONTENT_SIZE.get(args));
-                // TODO ee10 getServletHandler().setDecodeAmbiguousURIs(true);
+                getServletHandler().setDecodeAmbiguousURIs(true);
```

Comment:
https://github.com/jetty/jetty.project/pull/12791#issuecomment-2672247016 as documented in https://github.com/jetty/jetty-examples/blob/ff76268f75d7cc39945b0bf97059381bb5f27cc9/embedded/ee10-servlet-ambiguous-paths/src/main/java/examples/EmbedMe.java#L49.